WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1524,6 +1549,24 @@ static GSourceFuncs _handlerIntervention =
                 }
             }
 
+            _webProcessState = WebProcessCold;
+
+#ifdef WEBKIT_GLIB_API
+            if(webkit_web_view_get_is_web_process_responsive(_view) && _webProcessState == WebProcessCold)
+            {
+                _webProcessState = WebProcessHot;
+            }
+#else
+            WKPageIsWebProcessResponsive(
+                _page,
+                this,
+                [](bool isWebProcessResponsive, void* customdata) {
+                    WebKitImplementation* object = static_cast<WebKitImplementation*>(customdata);
+                   if (isWebProcessResponsive && object->_webProcessState == WebProcessCold)
+                        object->_webProcessState = WebProcessHot;
+                });
+#endif
```

Comment:
remove this. instead set state to hot in url changed.

Reply:
Removed this code.

Reply:
and set the webprocess state to hot.